nomos-services/membership/src/backends/mock.rs

```diff
@@ -313,6 +316,7 @@ mod tests {
         // (1st entry)
         // blocknumber 100 = 105 - k.historical_block_delta
         let result = backend.get_providers_at(service_type, 105).await.unwrap();
```

Comment:
I guess you can add `.1` directly here instead of re-defining the `result` variable?